src/Components/ExperimentTwo/FilmDescriptionPopUp.js

```diff
@@ -0,0 +1,16 @@
+import Button from "../ReUsableUIs/Button.js";
+import "../../CSS/ExperimentTwo/ModalStyle.css";
+
+const FilmDescriptionPopUp = (props) => {
+  return (
+    <div className="modal display-block">
+      <section className="modal-main">
+        <div className="CloseLayout">
```

Comment:
do you need all this nesting?

Reply:
Ah okay. So, I removed the inner <div>. Is that what you meant?